fw/http.c

```diff
@@ -7425,15 +7425,59 @@ __cfgop_brange_hndl(TfwCfgSpec *cs, TfwCfgEntry *ce, unsigned char *a)
 	return 0;
 }
 
-#define TFW_HTTP_CFG_CUSTOM_BRANGE(name)				\
+static int
+__cfgop_brange_parse_disallowed(char *str, unsigned char *a)
+{
+	char *start = str, *end = str;
+
+	while (true) {
+		if (*end == ' ' || *end == '\0') {
+			char* val = start;
+			unsigned long i0 = 0, i1 = 0;
+			*end = '\0';
+
+			if (tfw_cfg_parse_intvl(val, &i0, &i1)) {
+				T_ERR_NL("Cannot parse interval: '%s'\n", val);
+				return -EINVAL;
+			}
+			if (i0 > 255 || i1 > 255) {
+				T_ERR_NL("Too large interval bounds: '%s'\n", val);
+				return -EINVAL;
+			}
+
+			a[i0++] = 1;
+			printk(KERN_INFO "disallow: %s %d\n", str, a[i0-1]);
```

Comment:
Pleae use Tempesta FW's macros. I think this should be `T_DBG2` or `T_DBG3` (please check the description for which levels we use in which cases)